source/synthDrivers/sapi4.py

```diff
@@ -78,6 +83,42 @@ def IUnknown_Release(self, this, *args, **kwargs):
 		return super(SynthDriverBufSink, self).IUnknown_Release(this, *args, **kwargs)
 
 
+class SynthDriverSink(COMObject):
+	_com_interfaces_ = [ITTSNotifySinkW]
+
+	def __init__(self, synthRef: weakref.ReferenceType):
+		self.synthRef = synthRef
+		super(SynthDriverSink, self).__init__()
```

Comment:
This is more modern python

```suggestion
		super().__init__()
```